google/cloud/spanner/samples/samples.cc

```diff
@@ -989,6 +990,22 @@ void AddTimestampColumn(
 }
 // [END spanner_add_timestamp_column]
 
+//! [drop-column]
```

Comment:
comment: It looked suspicious that this tag is not referenced by a `@snippet`. I assume the tag is there in case we `Ctrl`+`F` the `drop-column` banner.

Reply:
I think(?) we have a few such non-devrel, non-snippet tags that are indeed only there to match their sub-command name, or otherwise delineate a block of code.

In this particular case it was only introduced so that I could adapt the pre-existing table-creation sample to the prerequisites of the proto-column samples.  If I recall correctly, the Java samples have "setup" and "teardown" code for each sample so that they are all independent, but we've evolved a model where we try to execute them in (some) sequence, so occasionally some between-sample adaptation is necessary.